amqp/amqp.go

```diff
@@ -84,6 +85,7 @@ func New(url string) (queue.Broker, error) {
 	}
 
 	b := &Broker{
+		mut:  sync.RWMutex{},
```

Comment:
since `mut` is not a pointer the zero value is already `sync.RWMutex{}`, so you can remove this line